server/lib/s3backup/__init__.py

```diff
@@ -189,6 +183,35 @@ def head_bucket(self, Bucket=None):
     def list_objects(self, **kwargs):
         return self.connector.list_objects(**kwargs)
 
+    # response decoding - tbh is the header returned
+    # by get_tarball_header().
+    @staticmethod
+    def s3_md5(tbh):
```

Comment:
How many places is `s3_md5` called from?  Seems like it is just one, so it is worth having a method for it?

Reply:
Two: pbench-backup-tarballs.py and header_md5() in the S3Config class. And even if there was only one, I'd want to hide what it does :-)